server.js

```diff
@@ -35,6 +35,10 @@ app.use(jsPath, express.static('./dist/client'));
 app.engine('hbs', exphbs({extname: '.hbs'}));
 app.set('view engine', 'hbs');
 
+const defaultPluginTitle = 'Jenkins Plugins';
+const defaultPluginDescription = 'Jenkins – an open source automation server which enables developers around the world to reliably build, test, and deploy their software';
```

Comment:
This description does not help in the case of plugins. I believe that almost all plugins already set a kind of description, but we might want to have a somethinf more explicit. Not a blocker

Reply:
The description comes from the index.jelly so it's required AFAIK, this is used on pages that aren't plugins, i.e. at least the home page.

Happy to reword it, but this is what it was before